scripts/performance/llm/pretrain_mixtral_8x7b.py

```diff
@@ -120,6 +120,9 @@ def override_recipe_configs(
         network='sharp' if args.use_sharp else None,
     )
 
+    if args.gpu.lower() in ['b200'] and "PYTORCH_CUDA_ALLOC_CONF" in executor.env_vars:
```

Comment:
Maybe add a comment like
`TODO: we currently disable PYTORCH_CUDA_ALLOC_CONF="expandable_segments:True" for mixtral_8x7b as it causes an unexpected error. This needs to be debugged.`